sklearn/calibration.py

```diff
@@ -207,6 +194,22 @@ def fit(self, X, y, sample_weight=None):
             calibrated_classifier.fit(X, y, sample_weight)
             self.calibrated_classifiers_.append(calibrated_classifier)
         else:
+            X, y = self._validate_data(
+                X, y, accept_sparse=['csc', 'csr', 'coo'],
+                force_all_finite=False, allow_nd=True
+            )
+
+            # Check that each cross-validation fold can have at least one
+            # example per class
+            n_folds = self.cv if isinstance(self.cv, int) \
```

Comment:
can you write an if/else statement without \

```python
if isinstance(self.cv, int):
    n_folds = self.cv
else:
    n_folds = self.cv_n_folds if hasattr(self.cv, "n_folds") else None
```

Reply:
`n_folds = self.cv_n_folds if hasattr(self.cv, "n_folds") else None` also doesn't fit on one line so I amended to `if, elif, else` but happy to change.